esphome/components/voice_kit/voice_kit.cpp

```diff
@@ -0,0 +1,59 @@
+#include "voice_kit.h"
+
+#include "esphome/core/defines.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace voice_kit {
+
+static const char *const TAG = "voice_kit";
+
+void VoiceKit::setup() {
+  ESP_LOGCONFIG(TAG, "Setting up Voice Kit...");
+
+  // Reset device using the reset pin
+  if (this->reset_pin_ != nullptr) {
```

Comment:
Its required so should never be nullptr

Reply:
[`0eeeeb3` (#57)](https://github.com/esphome/voice-kit/pull/57/commits/0eeeeb3fe2c76cedc982915677ab610aa1cecf9e)